### ISSUE TITLE
Android support

### ISSUE THREAD

**Issue by KCongUP** [Nethereum/Unity3dSimpleSampleNet461#10]
Does this project and libraries supported on Android?
I can build my project no problem and installed built file on my device, but when trying to send RPC request it return an exception.
exception:Error occurred when trying to send rpc requests(s): eth_call

**Comment by juanfranblanco** [comment]
Yes it does

**Comment by KCongUP** [comment]
Update: i have found out the problem was the Configuration - Scripting backend. If i use Mono and build, the function can run on device no problem. But if i switch to IL2CPP, it will thrown out the exception above.
Is there any way i can use these libraries with IL2CPP?

**Comment by juanfranblanco** [comment]
@KCongUP yes everything has been built to support AOT. Now the example includes all the libraries, so there will be some you won't need (reactive libraries, etc). If you use the subset of 351 you will be fine. And add as you need. https://github.com/Nethereum/Nethereum/releases/tag/4.1.0
